core/src/main/contraband/ClientAuth.contra

```diff
@@ -0,0 +1,8 @@
+package org.http4s.tls
```

Comment:
Would a `core/src/main/contraband/README.md` be worthwhile?  Or maybe an inline note about what happens with the `.contra` files?  I understand their value but it wasn't obvious and took some googling and several minutes of reading about Contraband before it clicked.